cookbooks/aws-parallelcluster-install/resources/ec2_udev_rules/partial/_debian_udev_configuration.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+
+action :set_udev_autoreload do
+  # allow udev to do network call
+  execute 'udev-daemon-reload' do
+    command 'udevadm control --reload'
+    action :nothing
```

Comment:
Is it the default action? If so, can we remove it?

Reply:
Yes, it is, and we can remove it, but in my opinion we should not. Leaving it makes clear that the command is not executed immediately

Reply:
To be honest, it is not clear... Maybe it's worth adding a comment.

Reply:
Added